src/vue/common/fees/fees-collection.js

```diff
@@ -7,15 +7,15 @@ const EMPTY_FEE = {
 }
 
 export class FeesCollection {
-  constructor ({ fees, asset, masterAccountId }) {
+  constructor ({ fees, assetCode, masterAccountId }) {
```

Comment:
masterAccountId - remove and check all place where used masterAccountId haw a constructor params